helm/templates/postgres-secrect.yaml

```diff
@@ -1,4 +1,5 @@
-{{- if and .Values.local.db.createSecret (not .Values.sharedData.db.secrets.useExternal) -}}
```

Comment:
Change the name of the file to "secret.yaml"

Reply:
done